CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Nothing should go in this section, please add to the latest unreleased version
   (and update the corresponding date), or add a new version.
 
-## [1.18.2] - 2022-08-01
+## [1.18.3] - 2022-08-01
```

Comment:
I believe this version should remain the same - 1.18.2 is not released yet

Reply:
The date should be updated though

Reply:
fixed